src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/Aes.cs

```diff
@@ -12,34 +10,20 @@ public abstract class Aes : SymmetricAlgorithm
     {
         protected Aes()
         {
-            this.BlockSize = 128;
-            this.KeySize = 256;
-            this.Mode = CipherMode.CBC;
-        }
-
-        public override KeySizes[] LegalBlockSizes
-        {
-            get
-            {
-                return s_legalBlockSizes.CloneKeySizesArray();
-            }
-        }
+            LegalBlockSizesValue = s_legalBlockSizes;
+            LegalKeySizesValue = s_legalKeySizes;
 
-        public override KeySizes[] LegalKeySizes
-        {
-            get
-            {
-                return s_legalKeySizes.CloneKeySizesArray();
-            }
+            BlockSizeValue = 128;
+            // TODO(12368): FeedbackSizeValue = 8;, but the field is not yet available.
+            KeySizeValue = 256;
+            ModeValue = CipherMode.CBC;
         }
 
         public static Aes Create()
         {
             return new AesImplementation();
         }
 
-
-
         private static readonly KeySizes[] s_legalBlockSizes = { new KeySizes(128, 128, 0) };
```

Comment:
@steveharter @stephentoub Addressed the protected field assignment concerns in a new commit.  So derived types that expect the field to be populated can read it, but writing to it only affects that one instance.  (NetFx bug is filed)